app/views/spree/admin/shared/_order_links.html.haml

```diff
@@ -19,3 +21,6 @@
               %span=link[:name]
 
 = render 'spree/admin/shared/custom-confirm'
+- if shipment_ready
```

Comment:
I am wondering if we could move the `shipment_ready` logic in an helper ?  so we could do `if order_shipment_ready?(@order)`

I am not a big fan of setting variable in the view, in this case it's not quite clear what's why `shipment_ready` is set to true

Reply:
Yes, that's a good idea.

Reply:
I agree, I might have been a bit hurry. I have put the logic in the order_helpers.rb